js/bigone.js

```diff
@@ -192,9 +192,13 @@ module.exports = class bigone extends Exchange {
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
             const symbol = base + '/' + quote;
+            const amountPrecisionString = this.safeString (market, 'base_scale');
+            const pricePrecisionString = this.safeString (market, 'quote_scale');
+            const amountLimit = (amountPrecisionString === undefined) ? undefined : '1e-' + amountPrecisionString;
```

Comment:
What if we introduce a helper base method for this ternary conditional to avoid code clutter?

Reply:
we could... I don't see it to be too critical

Reply:
Also, if precision is negative, it will yield `1e--...`.

Reply:
That's at least two reasons to detach this bit to a base method, what do you say?

Reply:
yeah that is a good point, will consider doing that this week